sessionctx/variable/sysvar.go

```diff
@@ -513,15 +512,14 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal | ScopeSession, "character_set_database", mysql.DefaultCharset},
 	{ScopeNone, "have_symlink", "YES"},
 	{ScopeGlobal | ScopeSession, "storage_engine", "InnoDB"},
-	{ScopeGlobal | ScopeSession, "sql_log_off", "OFF"},
-	{ScopeNone, "explicit_defaults_for_timestamp", "ON"},
+	{ScopeGlobal | ScopeSession, "sql_log_off", "0"},
+	{ScopeNone, "explicit_defaults_for_timestamp", "0"},
 	{ScopeNone, "performance_schema_events_waits_history_size", "10"},
 	{ScopeGlobal, "log_syslog_tag", ""},
 	{ScopeGlobal | ScopeSession, "tx_read_only", "0"},
 	{ScopeGlobal | ScopeSession, "transaction_read_only", "0"},
 	{ScopeGlobal, "rpl_semi_sync_master_wait_point", ""},
 	{ScopeGlobal, "innodb_undo_log_truncate", ""},
-	{ScopeNone, "simplified_binlog_gtid_recovery", "OFF"},
```

Comment:
Why remove this?

Reply:
`simplified_binlog_gtid_recovery` is renamed to `binlog_gtid_simple_recovery` in MySQL version 5.7.6.